test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/UpdateInstanceTests.cs

```diff
@@ -143,7 +143,7 @@ public async Task WhenUpdatingDicomMetadataWithExtendedQueryTagForASingleStudy_T
     {
         DicomTag ageTag = DicomTag.PatientAge;
         DicomTag patientSexTag = DicomTag.PatientSex;
-        string tagValue = "053Y";
```

Comment:
Extended query tag tests was using the same value so there is a probability that when tests run in parallel it could results in wrong value. So changing the value and also querying for specific study

Reply:
Update tests are not run in parallel but the different collections are run in parallel. I hope this should solve because I use specify study to validate.